static/script.js

```diff
@@ -96,15 +92,19 @@ function unfoldAll () { // eslint-disable-line no-unused-vars
     if (diffArray.length === 0) {
       diffArray = idsToUnfold
     }
-    diffArray.forEach(function (id) {
-      $('#' + id).toggleClass('show').parent().toggleClass('active-border')
-      id = id.replace('collapse', 'heading')
-      $('#' + id).toggleClass('unfold').toggleClass('collapsed')
-    })
+    expandRow(diffArray)
     sessionStorage.setItem('unfoldedRows', newSessionStorageValue)
   })
 
   $('.clickable').click(function () {
     $('#unfold-all').prop('checked', false)
   })
 }
+
+function expandRow (rowListID) {
```

Comment:
This is not really what I thought about and not the best practice.
Proper way is to extract anonymous function and apply it as callback to forEach.
```javascript
function expand (id) {
    $('#' + id).toggleClass('show').parent().toggleClass('active-border')
    id = id.replace('collapse', 'heading')
    $('#' + id).toggleClass('unfold').toggleClass('collapsed')
  }

diffArray.forEach(expand)
idsToUnfold.forEach(expand)
```